src/vconsole/vconsole-setup.c

```diff
@@ -53,6 +54,41 @@ static bool is_vconsole(int fd) {
         return ioctl(fd, TIOCLINUX, data) >= 0;
 }
 
+static int toggle_utf8(int fd, bool utf8) {
```

Comment:
Can do, but then diff tries to be clever and it's just a mess. Can squash them of course.